src/context/walletContext/walletContext.tsx

```
@@ -0,0 +1,303 @@
+                ...dbWallet,
+                connected: false    // this property will be processed into accurate values later in the flow
+            }))
+            setWallets(wallets)
+            setIsFetchingWallets(false)
+            setAreWalletsFetched(true)
+            return wallets
+        } else {
+            setIsFetchingWallets(false)
+            setAreWalletsFetched(false)
+            throwWalletError(interfaces.WalletErrorType.WALLET_FETCH_ERROR)
+        }
```

Biome

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)